recipes/example/meta.yaml

```diff
@@ -1,34 +1,44 @@
-{% set version = "1.0" %}
+# Jinja variables help maintain the recipe as you'll update the version only here.
+{% set version = "3.8.2" %}
 
 package:
-  name: example
-  version: {{ version }}
+    name: simplejson
+    version: {{ version }}
 
 source:
-  git_url: https://github.com/conda-forge/staged-recipes.git
-  git_tag: 24507f6c6b2d2e03caee550e13830df15d791751
+    # It is preferable to get the package "closer" to the source. For example use GitHub instead PyPI whenever possible.
+    fn: simplejson-{{ version }}.tar.gz
+    url: https://github.com/simplejson/simplejson/archive/v{{ version }}.tar.gz
+    # md5 is OK but sha256 is desirable.
+    sha256: 0041407316fbd35786883436c9ffaba4b799569174f22ee54e758f72249b1c67
 
 build:
-  number: 0
+    number: 0
+    # If the installation is complex, or different between Unix and Windows, use separate bld.bat and build.sh files instead of this key.
+    script: python setup.py install --single-version-externally-managed --record record.txt
 
 requirements:
-  build:
-    - python
-  run:
-    - python
+    build:
+        - python
+        # When setuptools is available add the `--single-version-externally-managed --record record.txt` above.
+        - setuptools
+    run:
+        - python
 
 test:
-  imports:
-    - example
-  commands:
-    - python -c "import example;"
+    # Some package might need a `test/commands` to check CLI.
+    imports:
+        - simplejson
+        - simplejson.tests
 
 about:
-  home: https://github.com/conda-forge/staged-recipes
-  license: BSD
-  summary: An example package
+    home: http://github.com/simplejson/simplejson
+    # Remember to specify the license variants for BSD, Apache, GPL, and LGLP.
+    license: MIT
+    summary: 'Simple, fast, extensible JSON encoder/decoder for Python'
 
 extra:
-  recipe-maintainers:
-    - HomerSimpson
-    - LukeSkywalker
+    recipe-maintainers:
+        # Always check with the people listed below if they are OK becoming maintainers of the recipe. (There will be spam!)
```

Comment:
We added it to the [docs]( https://github.com/conda-forge/conda-forge.github.io/blob/master/docs/guidelines.md ). I prefer docs for this than the wiki.

The only point I'm mentioning about ( https://github.com/conda-forge/staged-recipes/issues/139 ) is this clause.

> If they specify they only want to be contacted for certain recipes or none at all, please respect their wishes also add here what recipes (if any) they would like to be notified for. If they are no longer interested in any conda recipes, make that note here as well. Before contacting anyone please consult the list below to see if that contributor has restrictions.

So, all I'm saying, is people complain about over pinging on recipes should have their preferences listed there.